### DISCUSSION TITLE
Preset resolution not working.

### DISCUSSION THREAD

**Discussion by Animo78** [aclap-dev/video-downloadhelper#1209] [Help and Q&A]
### Your question

I been using your product for years.
It used to be in Chrome, now in Firefox.
I Notice in Firefox it doesn't remember the settings you made "Always prefer this quality"
Is this broken? Or how can I fix this?
Because I don't want to change every download to MP4 and 720p 

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"bdc3c66c5b50466d"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": false,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 19
}

**Comment by jcfrog** [comment]
Hi,
It seems it might be broken, yes. @paulrouget

**Reply by Animo78** [reply]
I also tested the BETA, this one has the same problem :(

**Reply by jcfrog** [reply]
I made some other tests on Youtube and it is actually working fine.
Could you share a sequence we could try to reproduce?

**Reply by Animo78** [reply]
> I made some other tests on Youtube and it is actually working fine. Could you share a sequence we could try to reproduce?

I send a video by the mail to you guys/gals

**Reply by GrampaWildWilly** [reply]
That is probably not helpful.  Maybe it is.  You didn't send whatever it is to me.  And you should not.  I don't want whatever you sent.  What I would want is the URL of a web page on which problematic content is hosted.  Not the name of a web site.  The URL of at least one page on that web site where you are having trouble.  If it's a site that requires a user ID & password, we have alternatives we can try.  But the starting point is a URL.  Don't send anything.  Just post the URL here & then we'll see if anybody can have the same problem downloading it that you are.
